core/include/edm/internal_spacepoint.hpp

```diff
@@ -18,12 +18,20 @@
 #include <vecmem/containers/jagged_vector.hpp>
 #include <vecmem/containers/vector.hpp>
 
+// std
+#include <algorithm>
+
 namespace traccc {
 
 struct neighbor_idx {
     size_t counts;
-    std::array<size_t, 9u> global_indices;
-    std::array<size_t, 9u> vector_indices;
+    /// global_indices: the global indices of neighbor bins provided by axis
+    /// class size of 9 is from (3 neighbors on z-axis) x (3 neighbors on
+    /// phi-axis)
+    size_t global_indices[9];
+    /// vector_indices: the actual indices of neighbor bins, which are used for
+    /// navigating internal_spacepoint_container
+    size_t vector_indices[9];
```

Comment:
Your suggestion might be better. But let us just leave it as it is. This edm might be deleted in the future because we may want to put `internal_spacepoint` into grid object, which is being developed in detray.

Reply:
Yeah I agree, let's leave this as it is.